compile.py

```diff
@@ -37,3 +37,12 @@
         report.write(f"## {label[10:]}\n\n")
         for content in issue_dict[label]:
             report.write(content.replace("\r\n", "\n"))
+
+with open("severity_counts.sh", "w") as counts_file:
+    total_count = 0
+    for label in SEVERITY_LABELS:
+        variable_name = label[10:].upper().replace(" ", "_") + "_COUNT="
+        count = len(issue_dict[label])
```

Comment:
If `label` does not exist in `issue_dict`, this would throw right? Can you add a check for this?